### ISSUE TITLE
net.forward() can't get the negative value of feature map

### ISSUE THREAD

**Issue by TXQGIT** [opencv/opencv#11884]
##### System information (version)

- OpenCV => 3.4.1
- Operating System / Platform => Ubuntu14.04
- Compiler => g++5.0

##### Detailed description

I am using the dnn module of OpenCV-3.4.1 with Caffe model. 
this is the prototxt file of my caffe model:
layer {
  name: "conv5"
  type: "Convolution"
  bottom: "conv4_relu"
  top: "conv5"
  param {
    lr_mult: 1
    decay_mult: 1
  }
  param {
    lr_mult: 2
    decay_mult: 0
  }
  convolution_param {
    num_output: 128
    pad: 1
    kernel_size: 3
  }
}

layer {
  bottom: "conv5"
  top: "conv5"
  name: "bn_conv5"
  type: "BatchNorm"
  batch_norm_param {
    use_global_stats: true
  }
}

layer {
  bottom: "conv5"
  top: "conv5"
  name: "scale_conv5"
  type: "Scale"
  scale_param {
    bias_term: true
  }
}

layer {
  name: "relu5"
  type: "ReLU"
  bottom: "conv5"
  top: "conv5_relu"
}

I want to get the feature map of conv5 layer(before ReLU operation).  So I use the code `Mat feats = net.forward("conv5")`. However the negative value  in feats become -0, while the positive value is unchanged. which means `Mat feats = net.forward("conv5")` and `Mat feats = net.forward("relu5")` get same results. 
In my application, these negative values are also important. Is there any way to get the negative value of feature map in Caffe model using OpenCV. Any advice will be great.

**Comment by dkurt** [comment]
@TXQGIT, it looks like a bug in fusion. May I ask you to check the results with `net.enableFusion(false)`?

see https://docs.opencv.org/master/db/d30/classcv_1_1dnn_1_1Net.html#af22e2f1a542d80ea797ba38902c726bd

**Comment by dkurt** [comment]
@TXQGIT, please check the changes from https://github.com/opencv/opencv/pull/11893.

**Comment by TXQGIT** [comment]
@dkurt, thanks for your help, I get the negative values of feature maps by adding `net.enableFusion(false)`.
